KarhooSDK/Service/Loyalty/LoyaltyEarn/KarhooLoyaltyEarnInteractor.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  KarhooLoyaltyEarnInteractor.swift
+//  KarhooSDK
+//
+//  Created by Edward Wilkins on 17/11/2021.
+//  Copyright © 2021 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class KarhooLoyaltyEarnInteractor: LoyaltyEarnInteractor {
+    
+    private let requestSender: RequestSender
+    private var identifier: String?
+    private var currency: String?
+    private var amount: Int?
+    private var points: Int?
+    
+    init(requestSender: RequestSender = KarhooRequestSender(httpClient: TokenRefreshingHttpClient.shared)) {
+        self.requestSender = requestSender
+    }
+    
+    func set(identifier: String, currency: String, amount: Int, points: Int) {
```

Comment:
If am not mistaken the points should be called `burnPoints ` and it is optional, and the default value is `0`

Reply:
Easy enough change